es/pages/best-coins-to-mine.html

```diff
@@ -163,6 +163,22 @@ <h2></h2>
                         </div>
                     </div>
 
+                    <div class="form-group">
```

Comment:
this too, we need to add it in `/pages/best-coins-to-mine.html` with special care of `src` path property `src="../img/ltc.svg"`

Reply:
done